charts/kyverno/values.yaml

```diff
@@ -109,6 +109,9 @@ config:
     #     - key: webhooks.kyverno.io/exclude
     #       operator: DoesNotExist
 
+  # -- Defines annotations to set on webhook configurations.
```

Comment:
Can we add a note about in what circumstances this would be used? Ex., AKS

Reply:
Done.